app/views/returns/NonExportedHumanMedicinesPlasticPackagingWeightView.scala.html

```diff
@@ -41,18 +42,16 @@
 
         @sectionHeader(messages("nonExportedHumanMedicinesPlasticPackagingWeight.caption"))
 
-        <h1 class="govuk-heading-l">@messages("nonExportedHumanMedicinesPlasticPackagingWeight.heading", amount.asKg)</h1>
```

Comment:
just spotted these will have different css classes now, do they look any different or is it just different ways to apply the same classes to a label vs h1

Reply:
some page that alreadt have the forId have the following html:

```
<h1 class="govuk-label-wrapper">
    <label class="govuk-label  govuk-label--l" for="value">
          content
    </label> 
</h1>
```

and they look ok to me.